ui/index.html

```diff
@@ -6,6 +6,6 @@
     <title>Geocoder</title>
   </head>
   <body>
-    <div id="root"></div>
```

Comment:
Just disambiguating between the document root (`html`) and the react "root" (renderer mount point).